src/typecheck/mod.rs

```diff
@@ -1996,7 +1993,7 @@ pub fn unify_erows(
                 }
                 (EnumRowsF::Extend { row: id, tail }, erows2 @ EnumRowsF::Extend { .. }) => {
                     let t2_tail = erows_add(state, &id, UnifEnumRows::Concrete(erows2))?;
-                    unify_erows(state, ctxt, *tail, t2_tail)
+                    unify_erows(state, *tail, t2_tail)
```

Comment:
The `&Context` argument was removed because it was useless: https://github.com/tweag/nickel/pull/793/files#diff-138af75d9d5a9700f6695900b1a0a4454d2acbfd263999a573553beae6ca2096L1980

Reply:
Ah, so your first clippy pass removed context, and then a snippet coming from the record rows refactoring still had it. I see, thanks!